src/entities/Badges/types.ts

```diff
@@ -0,0 +1,33 @@
+import camelCase from 'lodash/camelCase'
+
+import { capitalizeFirstLetter } from '../../helpers'
+
+export enum BadgeStatus {
+  BURNED = 'Burned',
```

Comment:
What I meant is that both key and properties are CamelCase with the upper first :P